tests/integration/ha/continuous_writes.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import subprocess
+import sys
+import time
+
+from kazoo.client import KazooClient
+from kazoo.retry import KazooRetry
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+
+
+def continous_writes(parent: str, hosts: str, username: str, password: str):
+    count = 0
+
+    while True:
+        client = KazooClient(
+            hosts=hosts,
+            timeout=10,
+            sasl_options={"mechanism": "DIGEST-MD5", "username": username, "password": password},
+            connection_retry=KazooRetry(max_tries=10, delay=1),
+            command_retry=KazooRetry(max_tries=10, delay=1),
+        )
+        client.start()
+
+        client.create(f"{parent}/{count}", acl=[], makepath=True)
+
+        client.stop()
+
+        time.sleep(1)
```

Comment:
**niptick** what about making the sleep customizable such that we can "tune the load" a bit

Reply:
Good idea!
Question, where would we tune it? As an argument in the tox command?

Reply:
possibly a fixture?

Reply:
I believe it makes sense to have the parameter in the function, such that - regardless of the where it is injected - even when importing this, one can tune the value of the tests. We can then use a fixture to tune this in tests. Of course, CI runs a fixed value, but we can easily tests to increase the load manually/locally, which I see it as a quite good thing. We are not yet doing any performance testing, but having an easy way to mock this is quite good